util/gcs/gcs.go

```diff
@@ -172,6 +172,9 @@ const (
 	DefaultACL = false
 	// PublicRead ACL for this upload.
 	PublicRead = true
+	// Default cache settings for this upload.
+	// Will cache, but only after verifying. See https://cloud.google.com/storage/docs/metadata#cache-control
+	DefaultCache = "no-cache"
```

Comment:
IMO `NoCache` is a better name for this. GCS does not support a customizable default cache setting.

`DefaultACL` is a specific GCS feature -- aka the set of ACLs to apply when not explicitly set by the upload. It is configurable too, see:

* `gsutil defacl`
* https://cloud.google.com/storage/docs/access-control/lists#default

Reply:
Ah, I see. Mixing these two senses of "default" is no good.

I don't like "NoCache" either, though; it's misleading. Might go with "CheckCache", "ValidateCache", or something to reflect how GCS acts.